src/ctelnet.h

```diff
@@ -121,8 +121,8 @@ const char OPT_COMPRESS2 = 86;
 const char OPT_MSP = 90;
 const char OPT_MXP = 91;
 const char OPT_102 = 102;
-const char OPT_ATCP = static_cast<char>(200);
-const char OPT_GMCP = static_cast<char>(201);
+const unsigned char OPT_ATCP = static_cast<unsigned char>(200);
+const unsigned char OPT_GMCP = static_cast<unsigned char>(201);
```

Comment:
This is one of those places where `auto` plays to good advantage - no need to redundantly duplicate a type declaration! :grin:
```suggestion
const auto char OPT_ATCP = static_cast<unsigned char>(200);
const auto char OPT_GMCP = static_cast<unsigned char>(201);
```